tests/functional/init/test_generate_project.py

```diff
@@ -19,7 +19,7 @@ def setUp(self):
         self.project_folder = os.path.abspath(
                                 os.path.join(self.output_dir, self.name))
         self.custom_location_folder = os.path.abspath(
-                                        os.path.join(self.output_dir, 'name-of-the-project'))
+                                        os.path.join(self.output_dir, 'Name of the project'))
```

Comment:
As we're now importing "pathlib2" for Python 2 and Python 3 comes with pathlib, does it make sense to create a new PR to replace all those `os.path` with the new format?

That will surely make the code cleaner, more readable and also better for Windows due to path joining, however I am from the opinion that this could be a separate PR as it'd need to be tested, adjust tests etc and this can slow down this rather important PR for Py3.

Reply:
Agree on both fronts here. We can have a clean up task to go through and update everything.